### PR TITLE
Automatically scroll the log to the bottom

### PR DESCRIPTION
**Pull request by dhensby**
When the deploy log is updated, we should scroll the div to the bottom to see the latest updates.

### CODE REVIEW

**Comment by stojg** [comment]
yes, I like this a lot! I'm going to see if I can get scrutinizer to behave properly by kicking it.

**Comment by dhensby** [comment]
this does kind of need testing...

**Comment by dhensby** [comment]
also, if the user manually scrolls we should probably turn it off, otherwise it'd get annoying

**Comment by stojg** [comment]
Yeah, we need to capture a user scroll event and then autoscroll. I think that was what I've tried doing 2 years ago or so, but it fell between the chairs.

File: javascript/deploynaut.js [hunk 1]
```diff
@@ -79,6 +79,8 @@
 			function(data) {
 				status.text(data.status);
 				content.text(data.content);
+                //scroll the content to the bottom
```
Comment (halkyon) [inline javascript/deploynaut.js]: We-ooh-wee-ohh. The indentation police are here to tell you that these spaces should be tabs ;-)

**Comment by dhensby** [comment]
@stojg done - If you scroll and you're not at the bottom, then it won't scroll for you. if you scroll to the bottom, it'll keep going to the bottom.

@halkyon fixed :P stupid IDE

**Comment by dhensby** [comment]
Also, this approach allows us to easily add a button to turn auto scrolling on/off :)

**Comment by dhensby** [comment]
Only other thing I can think is that the scroll event could be namespaced in case some other scroll event is attached and we accidently detach that with the `off` call

**Comment by dhensby** [comment]
Whoop

**Pull request merged**